pkg/daemon/gateway_linux.go

```diff
@@ -754,6 +767,134 @@ func (c *Controller) setIptables() error {
 	return nil
 }
 
+func (c *Controller) reconcileTProxyIPTableRules(protocol string, isDual bool) error {
+	if !c.config.EnableTProxy {
+		return nil
+	}
+
+	ipt := c.iptables[protocol]
+	tproxyPreRoutingRules := make([]util.IPTableRule, 0)
+	tproxyOutputRules := make([]util.IPTableRule, 0)
+	var probePorts []string
```

Comment:
We can use `strset.Set` to simplify the logic.

Reply:
1